src/App.tsx

```diff
@@ -53,6 +72,22 @@ const App = () => {
       >
         <Tab.Screen name="Home" component={HomeStack} options={{ headerShown: false }} />
         <Tab.Screen name="Contributors" component={Contributors} />
+        <Tab.Screen
```

Comment:
Added an extra tab for viewing the `Password Changed Screen` easier. Please advise if there is a different way to help preview screens @MelisaIm @lindentree

Reply:
The Password Changed Screen should appear when it needs to be used, we don't want it to appear as a tab in the UI. You can do this as part of your testing and leave a recording in the description of your ticket, but don't add it as a tab as part of your final PR :)

Reply:
I'll work on removing and adding a recording.